_includes/swc/setup.html

```diff
@@ -87,14 +87,15 @@ <h3>The Bash Shell</h3>
       </article>
       <article role="tabpanel" class="tab-pane active" id="shell-macos">
         <p>
-          The default shell in all versions of macOS is Bash, so no
-          need to install anything.  You access Bash from the Terminal
-          (found in
-          <code>/Applications/Utilities</code>).
+          The default shell prior to macOS 10.15 (Catalina) is Bash, and
```

Comment:
is what version people have common knowledge? Should we include a way to check the version or a date of the switch?

Reply:
Which version are you asking about?

Reply:
version of macOS, but actually, really the point is, how will learners know which instructions to follow?

Reply:
Apple Menu -> "About This Mac" tells you the current version.  But I think this is quite nit-picky, people can easily Google (other search engines are available) "what version of MacOS am I using".

For comparison, do we tell people how to determine which version of Windows they are running?

Reply:
For these installs, which version of windows someone is using doesn't change their instructions.  Since I don't use mac, if I were a helper or instructor, I wouldn't know how to check this and I expect I'm not alone.  Also, we run workshops in internet limited venues (where these instructions are downloaded and opened locally) at times and so it's better to not rely on people having to search for things; complete instructions is always better.

Reply:
I like the text from the shell lesson page because sometimes learners have, for various reasons, changed their default shell and may or may not know/remember they did. This solution captures both the MacOS update and the other case. 

> To check if your machine is set up to use something other than Bash, type `echo $SHELL` in your terminal window.

Reply:
Ahh, I understand why it's needed now - thanks!

I will amend the PR and add the same to the Linux instructions immediately below since it applies to both.

Reply:
I hope this is better?